src/Google.Maps/StaticMaps/StaticMapService.cs

```diff
@@ -77,11 +78,9 @@ public int GetMapToStream(StaticMapRequest mapOptions, System.IO.Stream outputSt
 
 			int totalBytes = 0;
 
-			WebRequest request = WebRequest.Create(requestUri);
-
-			using(WebResponse response = request.GetResponse())
+			using(HttpClient client = new HttpClient())
 			{
-				Stream inputStream = response.GetResponseStream();
+                Stream inputStream = client.GetStreamAsync(requestUri).Result;

```

Comment:
This will probably not work.  Probably retrieving Result property before its actually populated.  Will work when debugging line by line but highly likely wont work normally.